src/draw/sw/blend/helium/lv_blend_helium.h

```diff
@@ -0,0 +1,1345 @@
+{
+    arm_2d_size_t draw_size = {dsc->dest_w, dsc->dest_h};
+    int16_t des_stride = dsc->dest_stride / sizeof(uint16_t);
+    int16_t src_stride = dsc->src_stride / sizeof(uint16_t);
+    __arm_2d_impl_rgb565_tile_copy_opacity((uint16_t *)dsc->src_buf,
+                                           src_stride,
+                                           (uint16_t *)dsc->dest_buf,
+                                           des_stride,
+                                           &draw_size,
+                                           dsc->opa);
+    return LV_RESULT_OK;
+}
+
+static inline lv_result_t _lv_rgb565_blend_normal_to_rgb565_with_mask_helium(_lv_draw_sw_blend_image_dsc_t * dsc)
+{
+    arm_2d_size_t draw_size = {dsc->dest_w, dsc->dest_h};
+    int16_t des_stride = dsc->dest_stride / sizeof(uint16_t);
+    int16_t src_stride = dsc->src_stride / sizeof(uint16_t);
+    __arm_2d_impl_rgb565_src_msk_copy((uint16_t *)dsc->src_buf,
+                                      src_stride,
+                                      (uint8_t *)dsc->mask_buf,
+                                      dsc->mask_stride,
+                                      &draw_size,
+                                      (uint16_t *)dsc->dest_buf,
+                                      des_stride,
+                                      &draw_size);
+    return LV_RESULT_OK;
+}
+
+static inline lv_result_t _lv_rgb565_blend_normal_to_rgb565_mix_mask_opa_helium(_lv_draw_sw_blend_image_dsc_t * dsc)
+{
+    arm_2d_size_t draw_size = {dsc->dest_w, dsc->dest_h};
+    int16_t des_stride = dsc->dest_stride / sizeof(uint16_t);
+    int16_t src_stride = dsc->src_stride / sizeof(uint16_t);
+
+    __arm_2d_impl_gray8_colour_filling_with_opacity((uint8_t *)dsc->mask_buf,
+                                                    dsc->mask_stride,
+                                                    &draw_size,
+                                                    0x00,
+                                                    255 - dsc->opa);
+
+    __arm_2d_impl_rgb565_src_msk_copy((uint16_t *)dsc->src_buf,
+                                      src_stride,
+                                      (uint8_t *)dsc->mask_buf,
+                                      dsc->mask_stride,
+                                      &draw_size,
+                                      (uint16_t *)dsc->dest_buf,
+                                      des_stride,
+                                      &draw_size);
+    return LV_RESULT_OK;
+}
+
+static inline lv_result_t _lv_rgb888_blend_normal_to_rgb565_helium(_lv_draw_sw_blend_image_dsc_t * dsc,
+                                                                   uint32_t src_px_size)
+{
+    if(src_px_size == 3) {
+        return LV_RESULT_INVALID;
+    }
+
+    arm_2d_size_t draw_size = {dsc->dest_w, dsc->dest_h};
+    int16_t des_stride = dsc->dest_stride / sizeof(uint16_t);
+    int16_t src_stride = dsc->src_stride / sizeof(uint32_t);
```

Comment:
Sorry, what's the problem?

Reply:
Stride can be any number of bytes, for example 100x100 RGB888 with stride aligned up to 64, it cannot be converted to pixel stride, because it's not dividable by 3.

Reply:
as you can see, I only accelerate the condition where pixel size is 4. Even you align up to 64, there is no problem at all.

Reply:
@kisvegabor Any suggestion about this?

Reply:
Get it. It won't accelerate RGB888 because SIMD processes in 4 bytes.